src/EditorFeatures/VisualBasic/LineCommit/CommitFormatter.vb

```diff
@@ -32,8 +31,8 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
 
         <ImportingConstructor>
         <SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification:="Used in test code: https://github.com/dotnet/roslyn/issues/42814")>
-        Public Sub New(indentationManagerService As IIndentationManagerService)
-            _indentationManagerService = indentationManagerService
+        Public Sub New(inferredIndentationService As IInferredIndentationService)
```

Comment:
❕ Workspace services can't be directly imported to a constructor like this. You have to obtain them from a workspace instance.